src/renderer/page/rewards/view.jsx

```diff
@@ -6,12 +6,15 @@ import RewardTile from 'component/rewardTile';
 import Button from 'component/button';
 import Page from 'component/page';
 import classnames from 'classnames';
+import type { Reward } from 'types/reward';
 
 type Props = {
   doAuth: () => void,
+  fetchRewards: () => void,
   navigate: string => void,
   fetching: boolean,
-  rewards: Array<{ reward_type: boolean }>,
+  rewards: Array<Reward>,
+  claimed: Reward,
```

Comment:
I don't think `claimed` would be a reward right? ` { 'reward name': Reward } `, not super sure how to specify that in Flow